### PR TITLE
markup

### PR DESCRIPTION
**Pull request by Ihar777**

### CODE REVIEW

**Comment by pavelglebov** [comment]
👍 for reset css

**Comment by pavelglebov** [comment]
You added google font on your page by `<link>` tag, thats why you don't need to download font files manually and add them into repository (`fonts` folder). If you want to use local files (e.g. .ttf), you need to add them into css using `font-face`, and you don't need `<link>` tag in this case.

**Comment by pavelglebov** [comment]
Also, you don't need so many images in your `img` folder.

File: markup-2018q3/css/style.css [hunk 1]
```diff
@@ -0,0 +1,955 @@
+html{
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: correct styles is to write `html {`

File: markup-2018q3/css/style.css [hunk 2]
```diff
@@ -0,0 +1,955 @@
...
+  font-size: 1.6rem;
+  color: rgb(126, 126, 126);
+  background-color:rgb(255, 255, 255) ;
+    font-family: 'Lato', sans-serif;
+	width: 100%;
+	padding: 0;
+	margin-left: 0;
+	margin-right: 0;
+}
+#container {
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: an empty line is needed between rules

Comment (pavelglebov) [reply markup-2018q3/css/style.css]: Also, you should not apply styles to `id`, `id` should be used for JS manipulations.

File: markup-2018q3/css/style.css [hunk 3]
```diff
@@ -0,0 +1,955 @@
...
+
+img#logo{
+    max-width:100%;
+    height:auto;
+    position: absolute;
+    left: 15%;
+    top: 32px;
+}
+
+    .menu {
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: There are a lot of styling issues in file, you can check them with 1csslint`

File: markup-2018q3/index.html [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>markup</title>
+</head>
+<body>
+  <div id="container">
```
Comment (pavelglebov) [inline markup-2018q3/index.html]: I don't see any reason to have this container div here, you can use `body` instead.

File: markup-2018q3/css/style.css [hunk 5]
```diff
@@ -0,0 +1,955 @@
...
+  font-size: 15.96px;
+  color: rgb(255, 255, 255);
+  line-height: 1.2;
+  text-align: left;
+  width: 50%;
+  position: absolute;
+  left: 48.8%;
+  top: 35.906px;
+        
+  z-index: 9;
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: Almost half of properties in `.menu` class can be removed and nothing will change.

File: markup-2018q3/css/style.css [hunk 6]
```diff
@@ -0,0 +1,955 @@
...
+    display: block;
+    position: absolute;
+    top: 90%;
+    left: 0rem;
+    width: 360%;
+    height: 10%;
+    background-color: #ffffff;
+}
+
+.Header_Stick {
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: 1) class naming should always be lowercased and kebab-cased, e.g. `header-stick`. Also, you should read and apply BEM methodology for class namings.
2) Again, almost half on properties here can be removed and nothing will change. Also, this is not the best way to create a line (div with hr inside). Line can be created using border.

File: markup-2018q3/css/style.css [hunk 7]
```diff
@@ -0,0 +1,955 @@
...
+h1 {
+  font-size: 72px;
+  font-family: "Myriad Pro", sans-serif;
+  color: rgb(255, 255, 255);
+  font-weight: bold;
+  line-height: 1.2;
+  text-align: left;
+  position: absolute;
+  left: 41.4%;
+  top: 418.3px;
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: It is better to have not ideal proportions or not 100% pixel perfect layout, than using 'magic numbers' everywhere.

File: markup-2018q3/css/style.css [hunk 8]
```diff
@@ -0,0 +1,955 @@
...
+    content: ' ';
+    display: block;
+    position: absolute;
+    top: 0;
+    left: -40px;
+    width: 80px;
+    height: 120px;
+    transform: skew(-15deg, 0deg);
+    background-color: #2fac6c;
+    transform: skew(-15deg, 0deg);
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: Very nice! 👍 Transform is a nice way to do this skew.

File: markup-2018q3/index.html [hunk 9]
```diff
@@ -0,0 +1,362 @@
...
+                <td>$9.00 USD</td>
+                <td>Transfer Price No</td>
+            </tr>
+        </table>
+        </section>      
+         <section class="proposals">
+                <div class="enumprops">
+                    <div class= "basic">
+                       <div class="fa_anchor">
+                           <img src="img/anchor.png" class="pic" alt="anchor">
```
Comment (pavelglebov) [inline markup-2018q3/index.html]: You could use `font-awesome` instead of static pictures.

File: markup-2018q3/css/style.css [hunk 10]
```diff
@@ -0,0 +1,955 @@
...
+    margin-right: auto;
+}
+.heading{
+    margin-top: 18px;
+    margin-bottom: 18px;
+}
+
+.basic{
+    border-top: 6px solid #f9d423;
+    background-color: rgb(255, 255, 255);
```
Comment (pavelglebov) [inline markup-2018q3/css/style.css]: You should use `box-shadow` for these cards like in example.

**Comment by pavelglebov** [comment]
In general, the positioning of layout itself seems very fragile, it based on `absolute` elements, with static `heights`, `top` and `left` with magic numbers and so on.
I would suggest watching some videos on best practices of layout composing.

**Comment by pavelglebov** [comment]
**Timing:** **-30** one week late

**TT:** **-5**
1. Header.
- Interactive nav **+**
- Think of where h1 should be used **+**
- Interactive login/registe **+ -** (interactive, not not on full width of button)
2. Main.
- Interactive domain input with dropdown of first domains (.com, .net, .org). **+**
- Table should strictly be the same as design, and the cells number should be as much, as visible cells are. **-**
- The circle elements should be made using CSS, transformations. **-**
- The itmes should be intractive on hover. **+**
- links in "about us" should be interactive. **+**
- The feddback items should be interactive. (слайдер делать функциональным не обязательно, но желательно) **-**
- Services items should be links. **+**
3. Footer
- Menus should be intractive **+**
- Phone and mail interactions should correspond correctly. (например, при нажатии на телефон должно предложить звонок) **+**

**Code style, syntax**:  **-5** Css syntax has much to improve

**Design**: **-10** There are places that look differently, especially package cards.

**Responsiveness**: **-5** Some issues on resizing.

**Total: 45**

**Comment by pavelglebov** [comment]
Don't merge your pull request next time! Just create it and thats it!

**Comment by Ihar777** [comment]
Thank you, Pavel. I appreciate your taking the time to show me the issues in my pieces of code. I'll do my best to remember all of your programming tips and tricks.

    On Monday, November 19, 2018, 12:04:59 AM GMT+3, Pavel <notifications@github.com> wrote:  
 
 
Don't merge your pull request next time! Just create it and thats it!

**Pull request merged**